src/mudlet.cpp

```diff
@@ -3001,7 +3001,7 @@ void mudlet::slot_compact_input_line(const bool state)
         mpCurrentActiveHost->setCompactInputLine(state);
         // Make sure players don't get confused when accidentally hiding buttons.
         if (state && !mpCurrentActiveHost->mTutorialForCompactLineAlreadyShown) {
-            QKeySequence* shortcut = mShortcutsManager->getSequence(tr("Compact input line"));
+            QKeySequence* shortcut = mShortcutsManager->getSequence(qsl("Compact input line"));
```

Comment:
```suggestion
            QKeySequence* shortcut = mShortcutsManager->getSequence(qsl("Compact input line"));
            if (!shortcut) {
              qWarning() << __func__ << ": shortcut for compact input line not found";
              return;
            }
```

Just to prevent an accidental time-bomb. Didn't try compiling

Reply:
I have a bit different proposal.
Empty shorcut pointer would actually be fatal exception and really should not happen.
What can happen - is empty sequence, so means that user is clicking in menu - in that case warning is a bit strange.

Reply:
I've updated code, please tak a look.

Reply:
Even better!